nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/LogAttribute.java

```diff
@@ -123,12 +134,26 @@ protected void init(final ProcessorInitializationContext context) {
     protected String processFlowFile(final ProcessorLog logger, final DebugLevels logLevel, final FlowFile flowFile, final ProcessSession session, final ProcessContext context) {
         final Set<String> attributeKeys = getAttributesToLog(flowFile.getAttributes().keySet(), context);
         final ProcessorLog LOG = getLogger();
+        final String dashedLine;
+
+        String logPrefix = context.getProperty(LOG_PREFIX).getValue();
```

Comment:
An explicit call to evaluateAttributeExpressions is needed so that EL values are evaluated